mitosheet/src/mito/components/endo/utils.tsx

```diff
@@ -77,6 +77,21 @@ export const getTypeIdentifier = (columnDtype: string): string => {
     return typeText;
 }
 
+/**
+ * A helper function to check if a cell or column header matches a search value
+ * @param value - The value that you want to check against. This can be a cell value or a column header
+ * @param searchValue - The search value that you want to match
+ * @returns Boolean indicating if the value matches the search value
+ */
+export const checkMatchesSearch = (value: any, searchValue?: string) => {
+    if (searchValue === undefined || searchValue === '' || value === undefined) return false;
+    // Trailing spaces are ignored
+    searchValue = searchValue?.toLocaleLowerCase().trim();
```

Comment:
what is toLocaleLowerCase?

Reply:
It's supposed to be more friendly with other languages: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase. It'll output the same thing as `toLowerCase` most of the time